### PR TITLE
Java: Example

### PR DESCRIPTION
**Pull request by spinillos**
Java example

### CODE REVIEW

File: examples/java/CPU.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+import java.util.List;
+
+public class CPU {
+    public static Builder<Panel> cpuUsageTimeseries() {
+        String query = "((1 - sum without (mode) (rate(node_cpu_seconds_total{job=\"integrations/raspberrypi-node\", mode=~\"idle|iowait|steal\", instance=\"$instance\"}[$__rate_interval]))) " +
+                "/ ignoring(cpu) group_left " +
+                "count without (cpu, mode) (node_cpu_seconds_total{job=\"integrations/raspberrypi-node\", mode=\"idle\", instance=\"$instance\"}))";
+
+        Threshold th1 = new Threshold();
```
Comment (K-Phoen) [inline examples/java/CPU.java:17]: We should provide constructors for classes (not required now, an issue *for later*™ is fine)

File: examples/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                WithRow(new RowPanel.Builder("Network")).
+                WithPanel(Network.networkReceivedTimeseries()).
+                WithPanel(Network.networkTransmittedTimeseries()).
+                // Logs
+                WithRow(new RowPanel.Builder("Logs")).
+                WithPanel(Logs.errorsInSystemLogs()).
+                WithPanel(Logs.authLogs()).
+                WithPanel(Logs.kernelLogs()).
+                WithPanel(Logs.allSystemLogs()).
+                build();
```
Comment (K-Phoen) [inline examples/java/Main.java:45]: I'm a bit surprised to see the `build()` method in lowercase while other methods seem to all be UpperCamelCase. Is that on purpose? :thinking:

**Review by K-Phoen — APPROVED** [review]
What's the sdk-1.0.jar file? :thinking:

File: examples/java/Main.java (hunk 2 above)
Comment (spinillos) [reply examples/java/Main.java:45]: Nop, its a mistake. Sometimes I get confused with Go 😓

**Pull request merged**